google-cloud-logging/src/main/java/com/google/cloud/logging/spi/DefaultLoggingRpc.java

```diff
@@ -245,6 +250,7 @@ public void close() throws Exception {
     configApi.close();
     loggingApi.close();
     metricsApi.close();
+    providerManager.getChannel().shutdown();
```

Comment:
You can remove the close on the individual APIs now that you're using the provider manager - although, it doesn't hurt anything if they are called.